packages/frontend/src/router/routerUtils.ts

```diff
@@ -1,7 +1,21 @@
 import * as pathToRegexp from 'path-to-regexp';
 
-export function matchPath(currentPath: string, regexPathToMatch: string) {
-  const regexp = pathToRegexp(regexPathToMatch);
+export function matchPath(
+  currentPath: string,
+  regexPathToMatch: string | undefined,
+  exact: boolean = false,
```

Comment:
Use an options argument `{exact: true, strict: true}`

Reply:
@spion  - is now ok, or do you want to have an argument that is called explicitly "options" ?

Reply:
nope its all good